inspec/a2-iam-v2p1-only-integration/controls/get_authorized_projects.rb

```diff
@@ -89,6 +82,7 @@ def genInputProject(id, name)
               {
                 effect: "ALLOW",
                 # TODO this test will fail if there is a lone parameterized action (i.e. just iam:teams:get)
+                # ...what?! (@srenatus)
```

Comment:
hmmmm . . . seems worth investigating

Reply:
Yup! Didn't get to it today.

Reply:
https://github.com/chef/automate/pull/1109/commits/9b2105c124d702820b3674bbb7af447567ece5a1
not a problem. 🤷‍♂ I'll merge this.